applications/ParticleMechanicsApplication/custom_strategies/strategies/MPM_strategy.h

```diff
@@ -39,7 +39,6 @@
 // Custom includes
```

Comment:
this file is becoming huge, you shold consider refactoring it at some point

Reply:
Will put that in my note. I will do that after my thesis